sumologic-app-utils/src/sumoresource.py

```diff
@@ -625,26 +565,34 @@ def create_by_import_api(self, appname, source_params, *args, **kwargs):
             raise Exception("%s is available to Enterprise or Trial Account Type only." % appname)
 
         content = self._get_app_content(appname, source_params)
-        response = self.sumologic_cli.get_personal_folder()
-        personal_folder_id = response.json()['id']
-        app_folder_id = self._get_app_folder(content, personal_folder_id)
-        response = self.sumologic_cli.import_content(personal_folder_id, content, is_overwrite="true")
+
+        if "AWS Observability" in appname:
+            folder_id = self._create_or_fetch_quickstart_apps_parent_folder("Sumo Logic AWS Observability Apps ")
```

Comment:
Quick Start is present for folder naming. Similarly Observability is added for folder naming.

Reply:
Quickstart was removed as I can see in the diff can't we pass observability folder name as well. Asking this because this is a library(not just for aws observability) and should be generic.

Reply:
QuickStart is present but Yes, we can make it generic.

Reply:
quickstart is also present I thought you handled it using folder_prefix (see the githubdiff 605) seems then we'll have to create a backlog jira for this since it will require multiple changes.

Reply:
A new parameter as folder name should suffice, if it is present, then create that folder. I will track this in my tasks with AWS Observability

Reply:
cool then we should add this to observability improvement backlog jira so that it can tracked.